Classes/Issues/Issue+IssueType.swift

```diff
@@ -11,6 +11,18 @@ import IGListKit
 
 extension IssueOrPullRequestQuery.Data.Repository.IssueOrPullRequest.AsIssue: IssueType {
 
+    var id: String {
```

Comment:
These are needed because of what? 🤔

Reply:
also `isLocked` would be neat, but no blocker of course :)

Reply:
New Apollo version generated a bit different interface. I’ll take a look if it can generate the old one (I don’t have much experience w/ GraphQL and Apollo, but it’s a high time to get more)

Reply:
According to https://www.apollographql.com/docs/ios/fragments:
>  fields included through fragments are not exposed directly, but require you to access the data through the fragment explicitly

probably it was possible to access fields directly for single field fragments:
```
fragment nodeFields on Node {
  id
}
```